src/java/fr/paris/lutece/portal/business/file/ITemporaryFileDAO.java

```diff
@@ -86,4 +86,10 @@
      * @return the list of File
      */
     List<TemporaryFile> findByUser( AdminUser user );
+    
+    /**
+     * Select files older than the given number of days 
```

Comment:
Is days sufficiently granular ? Maybe you can split the parameter into a value and a `java.util.concurrent.TimeUnit`. See for instance https://github.com/lutece-platform/lutece-core/blob/dcb42f0ac5f9369104b5d72b6f175b81d0e94793/src/java/fr/paris/lutece/portal/service/mail/MailSenderDaemon.java#L86